doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,20 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
+
+The following metadata are not included in notifications by default, the `metadata` field (within
+`notification` field) in subscription is used to do so.
+
+* `previousValue`: the value and type of this metadata is the one of the attribute to which it
+  is attached previous to the processing of the request that triggered the notification. If the
+  type/value of `previousValue` is the same type/value than in the attribute in which it has been
+  included, then the attribute has not actually changed its value.
+
```

Comment:
the current state of implementations make this impossible because if there is no attribute value change no notification is sent ... but of course we are aiming at changing this in the near future